Calendar/BookingHelpers/BookingStaffHelpers.ps1

```diff
@@ -0,0 +1,54 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-MBPermissions {
+    param($identity)
+    # Get the Mailbox Permissions
+    $MBPermissions = Get-MailboxPermission -Identity $identity -ErrorAction SilentlyContinue
+    return $MBPermissions
+}
+
+function Get-MBRecipientPermissions {
+    param($identity)
+    # Get the Mailbox Recipient Permissions
+    $MBRecipientPermissions = Get-RecipientPermission -Identity $identity -ErrorAction SilentlyContinue
+    return $MBRecipientPermissions
+}
+
+function CheckMyBaseOptionsRBACRole($identity) {
+    $RBACRole = Get-ManagementRoleAssignment -RoleAssignee $identity -Role "MyBaseOptions" | Select-Object -Property RoleAssignee, Role
+    return $RBACRole
+}
+
+function CheckPersistedCapabilities($identity) {
+    $PC = Get-Mailbox -Identity $identity | Select-Object -ExpandProperty PersistedCapabilities
+    return $PC
+}
+
+function Get-GraphBookingsStaff {
+    param (
+        [string]$Identity
+    )
+
+    $MBstaff = Get-MgBookingBusinessStaffMember -BookingBusinessId $identity
+    $staff = @()
+    foreach ($staffMember in $MBstaff) {
+        $staff += [PSCustomObject]@{
+            Id                                       = $staffMember.Id
+            displayName                              = $staffMember.AdditionalProperties["displayName"]
```

Comment:
Confirming these are lowercase to match the Graph objects?

Reply:
confirmed, Graph uses Pascal Case